src/v3/src/hooks/useOnSubmit.ts

```diff
@@ -208,6 +208,7 @@ export const useOnSubmit = (): (options: OnSubmitHandlerOptions) => Promise<void
     // This way the flow can maintain the latest state handle. For eg. Device probe calls
     if (step === IDX_STEP.CANCEL_TRANSACTION) {
       SessionStorage.removeStateHandle();
+      removeUsernameCookie();
```

Comment:
Is this intentional? If the go back link is clicked in the device probe flow, you want to remove the username cookie?

Reply:
Bumping this comment, just making sure this is intentional?

Reply:
yeah, that's the intention